demuxer_test.go

```diff
@@ -3,14 +3,24 @@ package astits
 import (
 	"bytes"
 	"context"
+	"encoding/hex"
 	"fmt"
 	"io"
+	"strings"
 	"testing"
 
 	"github.com/asticode/go-astikit"
 	"github.com/stretchr/testify/assert"
 )
 
+func hexToBytes(in string) []byte {
```

Comment:
This is another nice way to test packets, I like it

Reply:
I might remove all whitespace in this helper so we can format the hex dumps with indentation

Reply:
done